bucket4j/src/main/java/com/linecorp/armeria/server/throttling/tokenbucket/TokenBucketBuilder.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.throttling.tokenbucket;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Builds a {@link TokenBucket} instance.
+ */
+public class TokenBucketBuilder {
+    private static final BandwidthLimit[] NO_BANDWIDTH_LIMITS = {};
+
+    private List<BandwidthLimit> limits = Collections.emptyList();
```

Comment:
Could we use `ImmutableList.Builder<BandwithLimit>` instead?

Reply:
`Collections.emptyList()` is immutable. Simply followed your contribution guidelines "Prefer using plain JDK API when the same behavior can be achieved with the same amount of code."
I can change it if you insist.

Reply:
Sorry, did not get your suggestion at first.
Fixed.

Reply:
No worries. Thanks for fixing. :smile: